README.md

```diff
@@ -1,2 +1,51 @@
 # Capstone-Project
-This is my first capstone project 
+
+First Capstone Project
+This is a project to implement an online website for a conference.
+
+Built With
+HTML
+CSS
+JavaScript
+Git and Github
+Getting Started
+Read down the page for instructions and requirements
+
+Prerequisites
+Code Editor(VS Studio Code) Web Browser (Google Chrome recommended)
+
+Live Demo
+Live Demo Link
+https://francis3400.github.io/Capstone-Project/
+
+How to Run
```

Comment:
- [ ] Let's personalize readne file to match what you did in the project. You an follow the given [template.](https://github.com/microverseinc/readme-template/blob/master/README.md)